directory.go

```diff
@@ -0,0 +1,40 @@
+package faker
+
+import (
+	"path/filepath"
+	"runtime"
+	"strings"
+)
+
+// Directory is a faker struct for Directory
+type Directory struct {
+	Faker *Faker
+}
+
+// Directory returns a fake directory path (the directory path style is dependent OS dependent)
+func (d Directory) Directory(levels int) string {
+	prefix := "/"
+
+	// This will only be true on Windows, coverage may be impacted depending
+	// on host OS
+	if runtime.GOOS == "windows" {
+		prefix = d.DriveLetter()
```

Comment:
You need to create a test that covers this line, you can create tests by not using the runtime.GOOS directly, but using an interface and mocking it. I did a similar think in [this](https://github.com/jaswdr/faker/blob/62506c31084e1a2fd33d67d578e1c22a07865c7d/utils.go#L14) file, where I create an interface that I used in [this](https://github.com/jaswdr/faker/blob/62506c31084e1a2fd33d67d578e1c22a07865c7d/lorem_flickr.go#L15) file and I mock it in [this](https://github.com/jaswdr/faker/blob/master/lorem_flickr_test.go#L39) test.